transformer.py

```diff
@@ -183,18 +197,18 @@ def add_parameters(parser: argparse.ArgumentParser) -> None:
                         default="Unknown",
                         help="name of the germplasm associated with the canopy cover")
 
-#pylint: disable=unused-argument
-def check_continue(transformer: transformer_class.Transformer, check_md: dict, transformer_md: dict, full_md: dict) -> tuple:
+
+# pylint: disable=unused-argument
+def check_continue(check_md: dict) -> tuple:
```

Comment:
The function signature has changed and is now different from everything else. Please restore the removed parameters